library/Makefile

```diff
@@ -1,8 +1,21 @@
 
 # Also see "include/mbedtls/config.h"
 
+ifeq ($(CC),iccarm)
+CFLAGS	?= --arm -Oh --silent
```

Comment:
If the defaul for none IAR is `-O2` , I think the default for IAR should be `-Om`. I believe `-Oh` is equivalent to `-O3`